src/lib.rs

```diff
@@ -452,6 +464,20 @@ fn set_pwm_channels_value_py(channels: Vec<PwmChannel>, value: u16) {
     }
 }
 
+#[cpy_fn_py]
+#[comment = "Like :py:func:`set_pwm_channel_duty_cycle`. This function sets the duty cycle for a list of multiple PWM channels.\n
+    Args:\n
+        channels ([:py:class:`PwmChannel`]): A list of PWM channels to configure.\n
+        duty_cycle (f32) : Duty cycle count value (0.0 : 1.0).\n
+    Examples:\n
+        You can use this method like :py:func:`set_pwm_channel_duty_cycle`.\n
+        >>> navigator.set_pwm_channels_value([PwmChannel.Ch1, PwmChannel.Ch16], 0.5)"]
+fn set_pwm_channels_duty_cycle_py(channels: Vec<PwmChannel>, duty_cycle: f32) {
+    for i in 0..channels.len() {
```

Comment:
wait, why not iterating over the vector ?>\
```suggestion
    for channel in channels {
```
?